.github/workflows/main.yaml

```diff
@@ -254,6 +254,7 @@ jobs:
         run: |
           export WASMER_DIR=`pwd`/package
           make test-integration
+          make test-integration-ios
```

Comment:
This probably need to be just when we are in the macOS environment

Reply:
The tests have a flag on them to prevent them running on Linux. Does that work?

Reply:
Yeah, I guess that works